src/test/java/org/jitsi/meet/test/DisableSelfViewTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright @ 2021 8x8 International
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test;
+
+import org.jitsi.meet.test.pageobjects.web.SettingsDialog;
+import org.jitsi.meet.test.util.TestUtils;
+import org.jitsi.meet.test.web.WebParticipant;
+import org.jitsi.meet.test.web.WebTestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.testng.annotations.Test;
+
+/**
+ * Tests the Breakout rooms functionality.
+ */
+public class DisableSelfViewTest
+    extends WebTestBase
+{
+    /**
+     * The local tile container xpath.
+     */
+    private final static String LOCAL_TILE_XPATH = "//span[@id='localVideoContainer']";
+
+    /**
+     * The local video menu trigger button xpath.
+     */
+    private final static String LOCAL_VIDEO_MENU_BUTTON_XPATH = LOCAL_TILE_XPATH + "//span[@class='localvideomenu']";
+
+    /**
+     * The local video menu hide self view button xpath.
+     */
+    private final static String HIDE_SELF_VIEW_BUTTON_XPATH = "//ul[@id='localVideoMenu']//a[@id='hideselfviewbutton']";
+
+    /**
+     * The participants.
+     */
+    private WebParticipant participant1;
+    private WebParticipant participant2;
+
+    @Override
+    public void setupClass()
+    {
+        super.setupClass();
+
+        ensureTwoParticipants();
+        participant1 = getParticipant1();
+        participant2 = getParticipant2();
+    }
+
+    @Test
+    public void testHideSelfViewFromMenu()
+    {
+        checkSelfViewDisplayed(false);
```

Comment:
Can you change the name of this function? It reads as it it checks if it's displayed, but the parameter is negated (hidden) so it doesn't read well in English.